frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -50,6 +57,26 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
     },
   } = React.useContext(AppContext);
   const tolerationSettings = notebookController?.notebookTolerationSettings;
+
+  let defaultStorageClass: StorageClassKind[] | undefined;
+
+  const allStorageClasses = async () => {
+    getStorageClasses().then((res) => {
```

Comment:
how can I resolve the promise to an array of storageclasses, which I can then filter on? Absolute newby, sorry